src/ez390.java

```diff
@@ -246,39 +247,55 @@
   /*
    * end of global ez390 class data and start of procs
    */
+
+
+
+   /**
+    * main is entry when executed from command line
+    * Create instance of ez390 class and pass
+    * parms to ez390 like z390 does.
+    * @param args - argument string - same as z390
+    */
 public static void main(String[] args) {
-  /*
-   * main is entry when executed from command line
-   * Create instance of ez390 class and pass
-   * parms to ez390 like z390 does.
-   */
-      ez390 pgm = new ez390();
-      pgm.process_ez390(args,null,null);
+
+  ez390 pgm = new ez390();
+  pgm.process_ez390(args,null,null);
 }
-public void process_ez390(String[] args,JTextArea log_text,JTextField command_text){
-   /*
-    *  execute 390 load module file passed as first arg
+
+
+
+   /**
+    * execute 390 load module file passed as first arg
     *
-    *  Note this may be called directly from z390 GUI or
-    *  from main when lz370 run from windows command line.
-    *  if called from main, the log_text balect will be null
-    *  and local put_log function will route to console instead
-    *  of the z390 log window.
+    * Note this may be called directly from z390 GUI or
+    * from main when lz370 run from windows command line.
+    * if called from main, the log_text balect will be null
+    * and local put_log function will route to console instead
+    * of the z390 log window.
+    * @param args - argument string (same as z390)
+    * @param log_text - 
+    * @param command_text - 
     */
-	init_ez390(args,log_text,command_text);
+public void process_ez390(String[] args,JTextArea log_text,JTextField command_text){
+
+    init_ez390(args,log_text,command_text);
     if (tz390.opt_ipl.length() > 0){
-    	run_pgm(pz390.zcvt_ipl_pgm);
-    	sz390.ez390_pgm = null;
-    	sz390.tot_link_stk = 0;
+        run_pgm(pz390.zcvt_ipl_pgm);
+        sz390.ez390_pgm = null;
+        sz390.tot_link_stk = 0;
     }
     run_pgm(pz390.zcvt_user_pgm);
     monitor_timer.stop();  // RPI 782
-	sz390.exit_ez390();
+    sz390.exit_ez390();
 }
-private void run_pgm(int zcvt_pgm_addr){
-	/*
+
+
+
+	/**
 	 * execute IPL pgm and/or application pgm
 	 */
+	 
+private void run_pgm(int zcvt_pgm_addr){
```

Comment:
Not sure whether one puts javadoc-style comments for a private method. If you do, add "@param zcvt_pgm_addr" in the above javadoc.